swap_meet/vendor.py

```diff
@@ -1,2 +1,89 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+        
+    def get_by_id(self, id):
+        for i in range (len(self.inventory)):
+            if self.inventory[i].id == id:
+                return self.inventory[i]
```

Comment:
In this situation, I think doing a `for` loop over each item in `self.inventory` (rather than indexing with `i` through a range) works just as well. Consider:

```python
        for item in self.inventory:
            if item.id == id:
                return item
```

This still passes the tests!